packages/flutter/lib/src/material/stepper.dart

```diff
@@ -124,6 +124,53 @@ class Step {
 /// to this widget based on some logic triggered by the three callbacks that it
 /// provides.
 ///
+/// {@tool sample --template=stateful_widget_scaffold_center}
+///
+///```dart
+///  int _index = 0;
+///  Widget build(BuildContext context) {
+///    return Container(
+///      height: 300,
+///      width: 300,
+///      child: Stepper(
+///        currentStep: _index,
+///        onStepCancel: (){
+///          if(_index <= 0 ) {return null;}
+///          setState(() {
+///            _index--;
+///          });
+///        },
+///        onStepContinue: (){
+///          if(_index >= 1) {return null;}
+///          setState(() {
+///            _index++;
+///          });
+///        },
+///        onStepTapped: (index){
+///          setState(() {
+///            _index = index;
+///          });
+///        },
```

Comment:
You could probably leave this out, since it's effectively not doing anything right?
```suggestion
```

Reply:
The `onStepTapped` method gets called when you tap on the Title of a step, it goes to the step. 
Suppose you're on Step 2 and press on the title of Step 1, it will jump to Step 1.